_posts/es/newsletters/2023-02-15-newsletter.md

```diff
@@ -0,0 +1,269 @@
+  más de 83 bytes de datos arbitrarios.  Razonaron que otros
+  métodos para almacenar grandes cantidades de datos se utilizan actualmente y
+  y no habría ningún daño adicional si se utilizara `OP_RETURN`
+  en su lugar.
+
+- **Dilución de tarifas en protocolos multipartitos:** Yuval Kogman
+[publicó][kogman dilution] en la lista de correo Bitcoin-Dev la
+descripción de un ataque contra ciertos protocolos multiparte.
+Aunque el ataque fue [descrito previamente][riard dilution],
+el post de Kogman atrajo de nuevo la atención sobre él.  Imaginemos que Mallory
+y Bob contribuyen cada uno con una entrada a una transacción conjunta con un
+tamaño y una comisión esperados, lo que implica un feerate esperado.  Bob proporciona un
+Bob proporciona un testigo del tamaño esperado para su aportación, pero Mallory proporciona un testigo mucho mayor de lo esperado.
+mayor de lo esperado.  Esto disminuye el feerate
+de la transacción.  Varias implicaciones de esto fueron discutidas en
+la lista de correo:
+
+- *Mallory consigue que Bob pague sus honorarios:* si Mallory tiene algún motivo ulterior
+  para incluir un testigo grande en la cadena de bloques, por ejemplo.
+  ejemplo, quiere añadir datos arbitrarios---puede usar parte de los honorarios de Bob para pagar
+  los honorarios por eso.  Por ejemplo, Bob quiere crear una transacción de 1.000 vbytes
+  vbyte con una tarifa de 10.000 satoshi, pagando 10 satoshi/vbyte para que se confirme rápidamente.
+  confirma rápidamente.  Mallory rellena la transacción con 9.000
+  vbytes de datos que Bob no esperaba, reduciendo su tarifa a 1
+  sat/vbyte.  Aunque Bob paga la misma tarifa absoluta en ambos casos,
+  no consigue lo que quería (confirmación rápida) y Mallory obtiene
+  9.000 sats de datos añadidos a la cadena de bloques sin coste alguno para ella.
+  para ella.
+
+- *Mallory puede ralentizar la confirmación:* una transacción con un
+  feerate puede confirmarse más lentamente.  En un protocolo sensible al tiempo, esto
+  podría causar un serio problema a Bob.  En otros casos, Bob podría
+  que Bob tenga que hacer un fee bump de la transacción, lo que le costará dinero adicional.
+  adicional.
+
+Kogman describe varias mitigaciones en su post, aunque todas ellas
+implican concesiones.  En un [segundo post][kogman dilution2], señala
+que no conoce ningún protocolo actualmente desplegado que sea
+vulnerable.
+
+- **Maleabilidad de la firma Tapscript:** en un aparte de la mencionada
+conversación sobre la dilución de tasas, el desarrollador Russell O'Connor
+[señaló][o'connor tsm] que las firmas para un [tapscript][tema
+tapscript] pueden aplicarse a una copia del tapscript colocada en otro lugar
+en el árbol taproot.  Por ejemplo, el mismo tapscript *A* se coloca en
+dos lugares diferentes en un árbol taproot.  Para utilizar la alternativa más profunda
+requerirá colocar un hash adicional de 32 bytes en los datos testigo de
+la transacción de gasto.
```

Comment:
Nit: the indentation is inconsistent here. Before, paragraphs were indented, here and below, the paragraphs are not indented.

I think @bitschmidty had also fixed some of that previously in his commit. The build appears to fail due to trailing whitespace, but I’m wondering whether it might be related to the indentation.